### ISSUE TITLE
Support for Scala 2.12.x

### ISSUE THREAD

**Issue by AndrewZurn** [vital-software/scala-redox#23]
Hello again,

I have been spending a little time updating our projects to the latest version of Play (and thus Scala 2.12 as well), and noticed that this library isn't being cross published to Scala 2.12.

Not sure the feasibility of doing that without digging around a little bit, but figured I'd ask/get it on the projects radar regardless.

When I get a little time, I can give updating the build file a go to see what happens (unless someone else is available or knows that it can/can't be done without a fair amount of effort).

Thanks,
Andrew

**Comment by apatzer** [comment]
Hey @AndrewZurn, happy to have Scala 2.12 supported. We likely won't move there for a bit of time, but Play/Slick/Akka all support it, so the migration hopefully shouldn't be too hairy. Happy for you to have a go at it!

**Comment by AndrewZurn** [comment]
Hey, forgot to follow up on this. I gave it a go a few weeks ago, but it doesn't look like json-annotation has been released on 2.12 yet, which will block this from being completed until then.

![image](https://user-images.githubusercontent.com/2126119/35456256-b47023d8-0289-11e8-99fb-8df86cbc4125.png)

**Comment by apatzer** [comment]
@AndrewZurn I've updated https://github.com/vital-software/json-annotation for scala 2.12 support
```
"com.github.vital-software" %% "json-annotation" % "0.4.0",
```
And updated v0.94 of `scala-redox` to use it. Hopefully this should unblock you.
